### PR TITLE
reduce 6lowpan traffic while provisioning constrained devices

### PR DESCRIPTION
**Pull request by rahul-daga-imgtec**
Instead of reading flowaccess object from constrained device, check whether flow access instance
has been registered on the local awa server to confirm if the device is provisioned or not.

Signed-off-by: Rahul Daga <Rahul.Daga@imgtec.com>

### CODE REVIEW

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

File: src/fdm_common.h [hunk 1]
```diff
@@ -169,4 +169,12 @@ typedef struct
     //! \}
 } Verification;
 
+
+
+typedef struct {
+    bool isDevicePresent;
```
Comment (pratik-prajapati-imgtec) [inline src/fdm_common.h]: Do the needful for doxygen here.

**Comment by abhijit-mahajani-imgtec** [comment]
Its generally working fine, but there are sometimes, when clicker console shows that FCAP is received but cloud has taken time to respond the status 4. so provisioning failure message is displayed in webpage.
I have passed timeout as 70sec to provision constrained device ubus call https://github.com/CreatorKit/webscripts/pull/7 , and seen good results.
Verified with webscripts change.

**Comment by abhijit-mahajani-imgtec** [comment]
Approved and Verified.

**Comment by pratik-prajapati-imgtec** [comment]
Approved

**Comment by nikhil-zinjurde-imgtec** [comment]
QA Verified

**Pull request merged**